### PR TITLE
Wands have an other behavior since the wand merge

### PR DESCRIPTION
**Pull request by syranez**
I have used texts from https://crawl.develz.org/trunk/changes.txt

### CODE REVIEW

File: crawl-ref/docs/crawl_manual.rst [hunk 1]
```diff
@@ -883,12 +883,10 @@ the 'q' command.
 ========================================
 
 Sometimes you will be lucky enough to find a stick which contains stored magical
-energies. Wands each have a certain amount of charges, and a wand will cease to
-function when its charges run out. You must identify a wand to find out how many
-uses it has left. This can be done with a scroll of identify; characters with a
-good Evocations skill may also deduce the number of charges simply upon evoking
```
Comment (NormalPerson7) [inline crawl-ref/docs/crawl_manual.rst]: Lines in the manual shouldn't be longer than 80 characters (as with all other areas of Crawl's code), and all four of these lines are slightly longer than that. Also, this sentence
```
Wands each have a certain amount of charges, which you immediately recognize as you pick it up.
```
doesn't make grammatical sense. Please rewrite it as
```
Wands each have a certain number of charges, which you immediately recognise when you pick them up.
```
Also note that, for historical reasons, Crawl follows Australian English spelling conventions, so "recognise" should be spelt with an "s".

Comment (syranez) [reply crawl-ref/docs/crawl_manual.rst]: Thank you.

I've made the necessary changes.

**Comment by MarvinPA** [comment]
Merged, thanks! I don't think you're listed in CREDITS.txt yet - what name would you like to be credited under?

**Comment by syranez** [comment]
I hardly did anything, so I do not need to be listed there... if you insist you can use "Michael Nowack".

Thanks for the merge!

**Pull request merged**